src/app/users/view-user/view-user.component.html

```
@@ -1,78 +1,105 @@
+<div
+  class="container m-b-20"
+  fxLayout="row"
+  fxLayout.lt-md="column"
+  fxLayoutAlign="end"
+  fxLayoutGap="2%"
+>
+  <button
+    mat-raised-button
+    color="primary"
+    (click)="editAppUser()"
+    matomoClick="Edit User Details"
```

HTMLHint (1.5.0)

[error] The attribute name of [ matomoClick ] must be in lowercase.

(attr-lowercase)